src/main/java/core/basesyntax/csv/reader/ReaderServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.csv.reader;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+
+public class ReaderServiceImpl implements ReaderService {
+    private final ListOfLinesMaker listOfLinesMaker;
+
+    public ReaderServiceImpl() {
+        listOfLinesMaker = new ListOfLinesMaker();
+    }
+
+    @Override
+    public List<String[]> readFromFile(String filePath) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(filePath))) {
+            return listOfLinesMaker.makeListOfLines(bufferedReader);
```

Comment:
Maybe it is better to return List of Strings, and let parser to split them, and do other operations. If so we can remove ListOfLinesMaker class.

Reply:
I totally agree
let's return `List<String>` from this method
and let parser do all further job